packages/sdk/src/get-generators/getGenerators.ts

```diff
@@ -120,6 +122,13 @@ export async function getGenerators(options: GetGeneratorOptions): Promise<Gener
     throw new Error(missingDatasource)
   }
 
+  config.datasources.forEach((datasource) => {
+    // If the value is one of the defaults, let's print a warning to the user
+    if (parseEnvValue(datasource.url) === defaultURL(datasource.provider)) {
```

Comment:
```suggestion
    // If the value is one of the defaults, let's print a warning to the user
    // we skip SQLite since the default is `file:./dev.db` and will work "out-of-the-box".
    if (datasource.provider !== 'sqlite' && parseEnvValue(datasource.url) === defaultURL(datasource.provider)) {
```
We should skip SQLite as it will work (it's just a file name, no username or password to set)